spec/services/to_form/base_mapper_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ToForm::BaseMapper do
+  describe '#call' do
+    let(:cocina_object) { build(:dro, title: title_fixture) }
+
+    it 'raises' do
+      expect { described_class.call(cocina_object:) }.to raise_error(NotImplementedError)
```

Comment:
This seems unnecessary to test.

Reply:
I was on the fence about this one. Good call.